apps/vast/vast-bridge.py

```diff
@@ -146,74 +124,97 @@ async def receive_intel(pub_endpoint, topic, intel_queue):
     socket.setsockopt(zmq.SUBSCRIBE, topic.encode())
     poller = zmq.Poller()
     poller.register(socket, zmq.POLLIN)
-    logger.info(f"Receiving intelligence items on {pub_endpoint}/{topic}")
+    logger.info(f"Receiving via ZMQ on topic {pub_endpoint}/{topic}")
     while True:
-        socks = dict(poller.poll(timeout=10))
+        socks = dict(poller.poll(timeout=100))
```

Comment:
The 10x change seems arbitrary. Can you add a comment explaining how you determined this value empirically? If it's not an implementation detail, you could consider making configurable via the configuration file.

Reply:
I added a comment.